pkg/reconciler/service/test.go

```diff
@@ -20,7 +21,7 @@ func (c *cleanup) removeKymaComponent(t *testing.T, version, component, namespac
 	t.Logf("Cleanup of component '%s' (version: %s, namespace: %s) started", component, version, namespace)
 
 	//render manifest
-	chartProv, err := c.reconciler.newChartProvider()
+	chartProv, err := c.reconciler.newChartProvider(&reconciler.Repo{})
```

Comment:
This looks weird. Just like you are passing an empty repository and than, the required information is appearing out of nowhere... What if the `newChartProvider` was left as was, but it had an additional method `WithRepo` or sth like that, instead?

e.g., `UsingRepo(*reconciler.Repo)`